src/main/java/org/fcrepo/importexport/ArgParser.java

```diff
@@ -189,13 +206,33 @@ protected Config parseConfiguration(final String[] args) {
             }
         }
 
+        validateConfig(config);
+
         // Write command line options to disk
         saveConfig(config);
 
         return config;
     }
 
     /**
+     * Validation routines for checking interdependent configuration parameters.
+     * @param config
+     */
+    private void validateConfig(final Config config) {
+        // verify bagit config
+        if (config.getBagProfile() != null &&
+                config.getBagConfigPath() == null) {
+            throw new RuntimeException("A bagit config path must be set when using a bagit profile.");
```

Comment:
Do we need to set a tag config file for every run? Could we see if the profile has required tags and then check to see if those requirements are met?